.circleci/config.yml

```diff
@@ -273,10 +275,8 @@ jobs:
       - run:
           name: e2e Tests
           command: |
-            source $HOME/.nvm/nvm.sh
-            nvm use
-            npm install -g yarn
-
+            node --version
+            yarn --version
```

Comment:
**nit:non-blocking** - I'm wondering if this was debugging or kept intentionally (perhaps to help debug future breakages)?

Reply:
Kinda both. I left it unintentionally, but I think it's useful enough to keep